### PR TITLE
Update dependencies.md

### PR DESCRIPTION
**Pull request by danbrotherston**
Added description of path option for git repository dependencies as this was added in https://github.com/dart-lang/pub/pull/1650 and it is no longer required for a package to be at the root of a repo.

### CODE REVIEW

**Comment by danbrotherston** [comment]
Signed the CLA.

File: src/tools/pub/dependencies.md [hunk 1]
```diff
@@ -174,6 +173,21 @@ The ref can be anything that Git allows to [identify a commit][commit].
 
 [commit]: http://www.kernel.org/pub/software/scm/git/docs/user-manual.html#naming-commits
 
+Pub assumes that the package is in the root of the git repository.  if this is
```
[the remark this answers is not in the document]
Comment (danbrotherston) [reply src/tools/pub/dependencies.md]: Done.

File: src/tools/pub/dependencies.md [hunk 2]
```diff
@@ -174,6 +173,21 @@ The ref can be anything that Git allows to [identify a commit][commit].
...
+not true, the path may be specified with the `path` argument:
```
[the remark this answers is not in the document]
Comment (danbrotherston) [reply src/tools/pub/dependencies.md]: Done.

**Comment by danbrotherston** [comment]
@kwalrath Made the small technical changes.

As for changing the PR...I am not entirely sure how to pull *only* this change into master, when I change the base of the PR, I end up pulling all the changes present in 1.x but missing in master into master.

Advice?  Is master being kept up to date with the changes in 1.x?

The changes are simple enough, I can simply create a new PR, if you don't mind losing the comment history.

FYI:  This is the PR (and base) that is created when you click the edit button in the documentation site.

File: src/tools/pub/dependencies.md [hunk 3]
```diff
@@ -174,6 +173,20 @@ The ref can be anything that Git allows to [identify a commit][commit].
 
 [commit]: http://www.kernel.org/pub/software/scm/git/docs/user-manual.html#naming-commits
 
+Pub assumes that the package is in the root of the Git repository.  To specify a different
```
[the remark this answers is not in the document]
Comment (danbrotherston) [reply src/tools/pub/dependencies.md]: My phrasing is original, I merely moved the sentence from one location to another.

[the remark this answers is not in the document]
Comment (chalin) [reply src/tools/pub/dependencies.md]: No worries, when making edits to a file we sometimes revise original content. I'll let @kwalrath weigh in.

File: src/tools/pub/dependencies.md [hunk 4]
```diff
@@ -174,6 +173,20 @@ The ref can be anything that Git allows to [identify a commit][commit].
...
+location in the repo use the `path` argument:
+
+{% prettify yaml %}
+dependencies:
+  kittens:
+    git:
+      url: git://github.com/munificent/kittens.git
+      ref: some-branch
+      path: path/to/my/plugin/
+{% endprettify %}
```
[the remark this answers is not in the document]
Comment (danbrotherston) [reply src/tools/pub/dependencies.md]: Done.

**Comment by danbrotherston** [comment]
I duplicated the pull request here:  https://github.com/dart-lang/site-www/pull/780

This one should now be declined.

**Pull request closed without merging**